addons/test_discuss_full/tests/test_performance.py

```diff
@@ -185,6 +186,7 @@ def _get_init_messaging_result(self):
                                     'is_company': False,
                                     'name': 'Ernest Employee',
                                     'out_of_office_date_end': False,
+                                    'write_date': self.users[0].partner_id.write_date.isoformat(),
```

Comment:
other dates in this test are asserted with `.strftime(DEFAULT_SERVER_DATETIME_FORMAT)`, what is the difference?

Reply:
I just need the date as a string in the payload. I don't really understand this comment.

Reply:
What I meant is that we typically don't use `isoformat()` so it is strange to start using it in this PR.

Date that are `read()` by the webclient as always returned in the format `.strftime(DEFAULT_SERVER_DATETIME_FORMAT)` which is `YYYY-MM-DD HH:mm:ss`.

We can argue about what is the best format, but it is strange that this PR introduces another return format, and I would suggest to stick to the usual way of doing it.

Or more simply `to_string()` on the `Datetime` field, I'm not sure why we're not always using that shortcut in our code.

Reply:
I wan't aware of the `.strftime`, if it's what we use most of time it make sens to keep doing that.